src/test/integration/README.md

```diff
@@ -1,6 +1,6 @@
 # Persons API Integration Tests
 
-The integration tests for Persons API.
+The integration tests for Persons API written using Python 2.
```

Comment:
But our dockerfile uses python 3? https://github.com/osu-mist/persons-api/blob/6272918587325953d38ec4b82809cf7287e1ae70/src/test/integration/Dockerfile#L1

Reply:
Oh you're right. For some reason I assumed it was Python 2. I'll change this.